src/models/stdlib/Stdlib.js

```diff
@@ -45,9 +50,12 @@ export default class Stdlib {
   _parseNameVersion(nameAndVersion) {
     const [name, version] = nameAndVersion.split('@')
     this.name = name
-    const packageVersion = this.getPackage().version;
-    this.version = version || packageVersion
+    this.version = version
     this.nameAndVersion = nameAndVersion
-    if (this.version !== packageVersion) throw Error(`Requested stdlib version ${version} does not match stdlib package version ${packageVersion}`)
+
+    const packageVersion = this.getPackage().version
+    if (!Stdlib.satisfies(packageVersion, version)) {
+      throw Error(`Requested stdlib version ${version} does not match stdlib package version ${packageVersion}`)
+    }
```

Comment:
Should we add a validation method besides `satisfies`? This is actually the logic we are needing here and in `StdlibProvider`